manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "author": "Steven A. Bjornson",
   "name": "Bandcamp Enhancement Suite",
-  "version": "1.0.4",
+  "version": "1.1.0",
```

Comment:
@emcniece good point, should be done before release and not as part of a feature. 

The unfortunate thing is that it does make it easier when developing since you can see the version number of the loaded plugin in the `chrome://extensions` menu.